bazelisk.py

```diff
@@ -273,6 +275,10 @@ def maybe_makedirs(path):
 
 
 def execute_bazel(bazel_path, argv):
+    # try to execute bazel wrapper in ./tools first.
+    if os.path.exists(TOOLS_BAZEL_PATH):
```

Comment:
I don't think we can assume that our current working directory is in the workspace root (it's allowed to run Bazel from any subdirectory of a workspace), so we should instead use find_workspace_root() to get the root directory and then use `os.path.join` to get to the path to $root/tools/bazel.